zyte_common_items/items.py

```diff
@@ -507,3 +514,107 @@ class ProductList(Item):
     #:
     #: See also ``canonicalUrl``.
     url: str = attrs.field(converter=url_to_str)
+
+
+@attrs.define(slots=True, kw_only=True)
+class BusinessPlace(Item):
+    """Business Place from a Google Maps Search."""
+
+    #: Unique identifier of the place on the website.
+    placeId: Optional[str] = None
+
+    #: The main URL that the place data was extracted from.
+    #:
+    #: The URL of the final response, after any redirects.
+    #:
+    #: Required attribute.
+    #:
+    #: In case there is no product data on the page or the page was not reached, the returned "empty"
+    #: item would still contain url field and metadata field with dateDownloaded.
+    #:
+    #: example: https://www.google.com/maps?cid=8273826543880430215
```

Comment:
```suggestion
```

Make sure you avoid using actual URLs where possible. Here an example is probably not needed, anyway.